server/plugin/create.go

```diff
@@ -32,14 +32,27 @@ func (p *Plugin) sendFileCreatedMessage(ctx context.Context, channelID, fileID,
 		return errors.Wrap(err, "failed to parse created time")
 	}
 	if shareInChannel {
+		userDisplayName := ""
+		userPhotoLink := ""
+		if len(file.Owners) != 0 {
+			userDisplayName = file.Owners[0].DisplayName
+			userPhotoLink = file.Owners[0].PhotoLink
+		} else {
+			user, err := p.Client.User.Get(userID)
+			if err != nil {
+				return errors.Wrap(err, "failed to fetch user")
+			}
+			userDisplayName = user.Username
```

Comment:
**question:** are we able to set the photo link here? Is it okay to leave it an empty string?